lib/util.js

```diff
@@ -15,6 +15,7 @@ const REGEX_LINK_HEADER = /\s*<([^>]*?)>\s*(?:;\s*(.*))?/;
 const REGEX_LINK_HEADER_PARAMS =
   /(.*?)=(?:(?:"([^"]*?)")|([^"]*?))\s*(?:(?:;\s*)|$)/g;
 
+// FIXME: conditinally support text/html
```

Comment:
With HTML support, we should include `text/html` and `application/ld+json`, but not sure best way to do that. It will also mess up some documentLoader tests?